app/index.js

```diff
@@ -1,5 +1,4 @@
-import React, { View, Text
-} from 'react-native';
+import React, { View, Text, Image } from 'react-native';
 
 import styles from './styles/default';
```

Comment:
**<a href='http://eslint.org/docs/rules/key-spacing'>key-spacing</a>**: Extra space before value for key "flex".